girder/girder_large_image/web_client/vue/components/FrameSelector.vue

```diff
@@ -170,6 +170,7 @@ export default Vue.extend({
                 :currentValue="indexInfo[index].current"
                 :valueMax="indexInfo[index].range"
                 :label="index.replace('Index', '')"
+                :sliderLabels="index.replace('Index', '') === 'C' ? imageMetadata.channels : []"
```

Comment:
Should we simplify this to just do `index === 'IndexC'` rather than doing the replace?